### PR TITLE
PWN-6779 - send screen UI fixes

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6779

## Description of Work

Send screen UI fixes

### CODE REVIEW

**Review by NikolaiEmelianov — APPROVED** [review]

File: app/src/main/res/layout/fragment_send_new.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
         app:layout_constraintTop_toTopOf="parent"
         app:navigationIcon="@drawable/ic_back_night"
         app:titleTextAppearance="@style/UiKit.TextAppearance.SemiBold.Title3"
+        app:titleTextColor="@color/text_night"
```
Comment (1sakov) [inline app/src/main/res/layout/fragment_send_new.xml:19]: Can you make a minor research, why the texts are not #2B2B2B by default? 
The text color is set inside `UiKit.TextAppearance`

Comment (kabramovp2p) [reply app/src/main/res/layout/fragment_send_new.xml:19]: we had an agreement that the color is affixed separately as it changes and it was not recorded in the style

Comment (gslevinkov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: i can confirm that
we can make it default and override it

Comment (kabramovp2p) [reply app/src/main/res/layout/fragment_send_new.xml:19]: ok let's do that
 i will

Comment (kabramovp2p) [reply app/src/main/res/layout/fragment_send_new.xml:19]: heh lol
it is already there
but
here we have 
app:titleTextColor="@color/text_night"

Comment (kabramovp2p) [reply app/src/main/res/layout/fragment_send_new.xml:19]: and here is just 
image
       <item name="android:textColor">@color/text_night</item>

Comment (kabramovp2p) [reply app/src/main/res/layout/fragment_send_new.xml:19]: @1sakov @gslevinkov ^^

Comment (1sakov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: @kabramovp2p textcolor is still not working in textviews.

Comment (1sakov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: @gslevinkov let's make the default color `text_night`
If there is a different text, we will override in xml directly

Comment (gslevinkov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: separate task or..?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/res/layout/fragment_send_new.xml (hunk 1 above)
Comment (1sakov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: yes separate. 
@kabramovp2p you can take it if you want

**Review by 1sakov — APPROVED** [review]

File: app/src/main/res/layout/fragment_send_new.xml (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/res/layout/fragment_send_new.xml:19]: https://p2pvalidator.atlassian.net/browse/PWN-6848

**Pull request merged**